test/e2e_node/e2e_node_suite_test.go

```diff
@@ -118,3 +124,16 @@ func (lr *LogReporter) SpecSuiteDidEnd(summary *types.SuiteSummary) {
 	b.WriteString("******************************************************\n")
 	glog.Infof(b.String())
 }
+
+func maskLocksmithdOnCoreos() {
+	data, err := ioutil.ReadFile("/etc/os-release")
+	if err != nil {
+		glog.Fatalf("Could not read /etc/os-release: %v", err)
+	}
+	if bytes.Contains(data, []byte("ID=coreos")) {
+		if output, err := exec.Command("sudo", "systemctl", "mask", "--now", "locksmithd").CombinedOutput(); err != nil {
+			glog.Fatalf("Could not mask locksmithd: %v, output: %q", err, string(output))
+		}
+	}
+	glog.Infof("Locksmithd is masked successfully")
```

Comment:
I see it in the [node e2e logs](https://storage.cloud.google.com/kubernetes-jenkins/pr-logs/pull/25004/node-pull-build-e2e-test/3847/build-log.txt)

>
Success Finished Host tmp-node-e2e-54dd96d6-e2e-node-coreos-stable20160218-image Test Suite
Warning: Permanently added 'tmp-node-e2e-54dd96d6-e2e-node-coreos-stable20160218-image'
...
I0502 07:07:00.610062     911 e2e_node_suite_test.go:101] ******************************************************
I0502 07:07:00.774986     911 e2e_node_suite_test.go:138] Locksmithd is masked successfully